### ISSUE TITLE
Unable to fully delete a full page chat room

### ISSUE THREAD

**Issue by rysherz** [gdpelican/babble#212]
The steps to reproduce:
1. git clone -b beta https://github.com/gdpelican/babble.git
2. Create a full page chat room based on category permissions
3. Post any message to it
4. Delete this chat room (Settings -> Chats -> Select chat room -> Click Delete)

Result: 'Chats' link is still on the category page, but the chat room can't be loaded. New chat room for this category can't be created because of "Unable to save that chat channel. Please try again.", but there is no such chat room in the list.

**Comment by odysseywestra** [comment]
I'm also being affect by this bug. Is there a way to fix it?

**Comment by gdpelican** [comment]
This should be fixed in the latest beta branch, v2.0.2

**Comment by gdpelican** [comment]
In order to fix this for existing chat channels, run the following commands in your rails console:

```
c = Category.find_by(name: "<category name>")
c.custom_fields['chat_topic_id'] = nil
c.save
```
